src/YOUR_NAME_ex/ex02/file_stats.py

```diff
@@ -1,15 +1,24 @@
-#!/usr/bin/env python3
-# -*- coding: utf-8 -*-
-"""
-Created on Fri Sep 27 08:19:09 2019
+""" This module is part of the ex_02 assignment """
 
-@author: jorgenkongsro
-"""
-def char_counts(textfilename):
-    """ function docstrin
+
+def char_counts(filename):
+    """ opens the file with the given 'filename' using encoding 'utf-8' and
+    reads the entire file content into a single string the file and count
+    how often each character code (0–255) occurs in the string
+
+    Returns
+    -------
+    result: list or tuple
     """
-    
-    return
+    check_string = open(filename, encoding='utf8').read()
```

Comment:
looks like you started this file before opening the ex_02 branch. Please don't do this in the future.